src/containers/Profile/index.test.js

```diff
@@ -1,24 +1,137 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import { Profile } from './index';
+import { Profile, mapDispatchToProps } from './index';
+import * as thunk from '../../thunks/createUser';
 
 describe('Profile', () => {
   describe('Profile component', () => {
     let wrapper;
+    const mockCreateUser = jest.fn();
     const mockProps = {
-      createUser: jest.fn(),
+      createUser: mockCreateUser,
       email: 'Hill@gmail.com',
       firebaseID: 'bAzN8PSfQSPL9rg4pMhTha8zewn2',
       image: 'https://avatars1.githubusercontent.com/u/49459620?v=4',
       name: 'Hillary'
     };
-
+    
     beforeEach(() => {
       wrapper = shallow(<Profile {...mockProps} />);
+      window.fetch = jest.fn();
     });
 
     it('should match the snapshot', () => {
       expect(wrapper).toMatchSnapshot();
     });
+
+    it('should have a default state', () => {
+      const expected = {
+        name: mockProps.name,
+        email: mockProps.email,
+        program: '',
+        module: '',
+        pronouns: '',
+        slack: '',
+        skill1: '',
+        skill2: '',
+        availabilities: [
+          false,
+          false,
+          false,
+          false,
```

Comment:
A shortcut you could use for an array with all the same values is Array(`length`).fill(`value`). Don't worry about changing this now though. It's late and this project is due tomorrow. 😴